README.md

```diff
@@ -50,69 +48,42 @@ The Rollbar client for EmberJS applications.
     return ENV;
   }
 ```
-
-## Usage
-
-### Rollbar Service
-In your component, controller, route, object (or whatever) you can inject the `rollbar` service, eg:
-
+3. Initialize and start the Rollbar in `/app/app.js` file:
 ```js
-import Ember from 'ember';
-const { Component, inject } = Ember;
+import config from 'YOUR_APP/config/environment';
+import { startRollbar } from 'ember-rollbar-client';
 
-export default Component.extend({
-  rollbar: inject.service()
```

Comment:
What are your thoughts on keeping the service around? I'm purely thinking from a backwards compatibility point of view

We use the service a fair bit.. Even if it just wrapped the rollbar instance in some way?

Reply:
The goal is to unify the API with Rollbar. Get rid of any abstraction where possible. I realize this is a huge breaking change but it's a big step for simplification and not depending on the custom implementation.

The migration should be quite easy. In most cases, it would be replacing the
`@service rollbar` with `import { rollbar } from 'ember-rollbar-client'`

But will think a bit more if actually still having the service wouldn't be a great way for easier migration...

Reply:
I think the main benefit of having the service is it can be easily stubbed/replaced while testing. Doing the same when importing a module is a little more difficult.

The alternative is for apps to provide their own service that wraps `ember-rollbar-client`, which is probably a better approach when it comes to logging anyway 👀

Reply:
Stubbing can be still easy if you use a tool like sinon :)

Wrapping every lib into service is in my opinion not a good pattern. That adds up additional layers of abstraction without a need.

The client app could create its own service like `logger` (which should be not related to the provider) and implement an interface that suits its needs. That's called a wrapper pattern.
However, exposing a service through the addon will have no value in comparison to the ES6 module

Reply:
Yes, I think we'll probably go with that approach. I'm just trying to avoid having all our usages of `@service rollbar` in our apps and addons having to change as part of this upgrade.

> And Ideally, you should never stub the logger ;)

Do you have any suggestions about what we should do instead? I did a bit of research, the best I found was creating a `logger` service, and to switch it out during testing

Reply:
The rollbar is by default disabled in tests :) you can manage its state by configuring the `enabled` option. Also with new API, there is this `stopRollbar()` function to better control ember integration. There should be no need to stub the logger :)